tests/p-chain-tests.js

```diff
@@ -255,17 +255,17 @@ describe('Staking tests', async function () {
     const pathPrefix = "44'/9000'/0'";
     const pathSuffixes = ["0/0", "0/1", "100/100"];
     const ui = await flowMultiPrompt(this.speculos, [
-      [{header: 'Sign', body: 'Add Validator'}],
-      [{header: 'Transfer', body: '3.999 AVAX to local1mg47uqd7stkvqrp57ds7m28txra45u2uzkta8n'}],
-      [{header: 'Validator', body: 'NodeID-NFBbbJ4qCmNaCzeW7sxErhvWqvEQMnYcN' }],
-      [{header: 'Start time', body: '2020-07-29 22:07:25 UTC' }],
-      [{header: 'End time', body: '2020-08-28 21:57:26 UTC' }],
-      [{header: 'Total Stake', body: '2000 AVAX' }],
-      [{header: 'Stake',body: '2000 AVAX to local18jma8ppw3nhx5r4ap8clazz0dps7rv5u00z96u'}],
-      [{header: 'Rewards To', body: 'local1mg47uqd7stkvqrp57ds7m28txra45u2uzkta8n' }],
-      [{header: 'Delegation Fee', body: '0.01%' }],
-      [{header: 'Fee',body: '0.001 AVAX'}],
-      [{header: 'Finalize',body: 'Transaction'}],
+      {header: 'Sign', body: 'Add Validator'},
+      {header: 'Transfer', body: '3.999 AVAX to local1mg47uqd7stkvqrp57ds7m28txra45u2uzkta8n'},
+      {header: 'Validator', body: 'NodeID-NFBbbJ4qCmNaCzeW7sxErhvWqvEQMnYcN' },
+      {header: 'Start time', body: '2020-07-29 22:07:25 UTC' },
+      {header: 'End time', body: '2020-08-28 21:57:26 UTC' },
+      {header: 'Total Stake', body: '2000 AVAX' },
+      {header: 'Stake',body: '2000 AVAX to local18jma8ppw3nhx5r4ap8clazz0dps7rv5u00z96u'},
+      {header: 'Rewards To', body: 'local1mg47uqd7stkvqrp57ds7m28txra45u2uzkta8n' },
+      {header: 'Delegation Fee', body: '0.01%' },
+      {header: 'Fee',body: '0.001 AVAX'},
+      {header: 'Finalize',body: 'Transaction'},
     ]);
```

Comment:
This is beautiful and exactly what we're trying to move towards

Reply:
Yeah that's your code  changes :) I will have to group those into N sized chunks based memory constraints (can go larger than the 4 I started with, but there will be a limit)